index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <title>ffmprovisr</title>
-    <meta charset="utf-8">
+    <meta name="viewport" content="text/html, charset=UTF-8, width=device-width, initial-scale=1">
```

Comment:
Is this widely supported?

Reply:
Yeah. Mostly this sets the default sizing so they can change as needed to fit onto different-sized devices. In this branch, only the h1 tag changes size based on how big the window is so it doesn't get smushed.

Reply:
Hmm… not sure. On Safari it gives now:

<img width="1189" alt="bildschirmfoto 2017-10-09 um 15 40 35" src="https://user-images.githubusercontent.com/3246607/31340986-410e6732-ad08-11e7-9855-9d1a6f0f52c4.png">

Reply:
It sets the standard charset of the browser, and does not understand to change. Putting again back that line, gives:

<img width="1187" alt="bildschirmfoto 2017-10-09 um 15 42 47" src="https://user-images.githubusercontent.com/3246607/31341065-89780bd6-ad08-11e7-955a-849df8bc1e5d.png">

Reply:
Arg, thanks, hmm... I mean the charset=utf-8 *is* still in there. It might need an apostrophe instead of a comma. One sec!

Reply:
Sorry about that, I totally missed that charset="" was not scoped within the content=""!